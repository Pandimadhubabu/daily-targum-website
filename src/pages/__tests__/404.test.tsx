jest.mock('../../components/Theme');

import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../404.page';

describe("<404 />", () => {

  it("renders correctly", () => {
    const tree = renderer.create(
      <NotFound/>
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
  
});