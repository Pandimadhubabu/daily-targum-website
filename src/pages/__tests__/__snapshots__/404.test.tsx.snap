// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<404 /> renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "paddingBottom": 40,
      "paddingLeft": 10,
      "paddingRight": 10,
      "paddingTop": 40,
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "maxWidth": "calc(1000px + 22vw)",
        "overflowX": "hidden",
        "width": "100%",
      }
    }
  >
    <div
      style={
        Object {
          "alignItems": "flex-start",
          "display": "flex",
          "flexDirection": "column",
        }
      }
    >
      <h1
        style={
          Object {
            "fontSize": "2.5rem",
            "fontWeight": 800,
            "lineHeight": "1.2em",
            "marginBottom": 10,
          }
        }
      >
        404. Not found.
      </h1>
      <p
        style={
          Object {
            "fontSize": "1rem",
            "hyphens": "auto",
            "lineHeight": "1.8em",
            "marginBottom": 10,
            "wordWrap": "break-word",
          }
        }
      >
        Sorry, but the page you were trying to view does not exist.
      </p>
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        <div
          style={
            Object {
              "backgroundColor": "var(--accent-color)",
              "borderRadius": 5,
              "color": "#fff",
              "cursor": "pointer",
              "fontSize": "1.3rem",
              "padding": 10,
              "paddingLeft": 40,
              "paddingRight": 40,
            }
          }
        >
          Go to Home
        </div>
      </a>
    </div>
  </div>
</div>
`;
